### PR TITLE
Add test for checking delete button state when prohibiting deleting in share

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Add acceptance test to check that the delete button is disabled when deletion is prohibeted during sharing.

## Related Issue

- Related to #1614

## How Has This Been Tested?

:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js [hunk 1]
```diff
@@ -301,6 +301,36 @@ module.exports = {
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementPresent(this.getFileRowSelectorByFileName(element))
     },
+    assertActionDisabled: function (action, fileName) {
+      const btnSelectorHighResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[1]'
+      const btnSelectorLowResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[last()]'
+      const fileActionsBtnSelector = this.getFileRowSelectorByFileName(fileName) +
+        this.elements.fileActionsButtonInFileRow.selector
```
Comment (individual-it) [inline tests/acceptance/pageObjects/FilesPageElement/filesList.js]: can we put this and https://github.com/owncloud/phoenix/blob/00a9f009e7af1eacda0c6d195ee91298b4888508/tests/acceptance/pageObjects/FilesPageElement/filesList.js#L16 into one function?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5504/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5504/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5504/

<p>

















































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5512/

<p>

































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5512/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5512/

<p>







</p></details>

**Pull request merged**